AUTHORS

```diff
@@ -20,3 +20,5 @@ stefan.schippers@gmail.com (Stefan Schippers)
 amro_tork@mabrains.com (Amro Tork)
 farag_agoor@mabrains.com (Farag Agoor)
 mohanad_mohamed@mabrains.com (Mohanad Mohammed)
+rehab_hassan@mabrains.com (Rehab Hassan)
```

Comment:
Extra new line at end of file.

Reply:
Resolved at: https://github.com/google/globalfoundries-pdk-libs-gf180mcu_fd_pv/pull/1/commits/29982d051040d7d15a8070bef903eb32f77a12fd

Reply:
LGTM.